interfaces/builtin/network_manager.go

```diff
@@ -70,11 +70,10 @@ network packet,
 /run/udev/data/* r,
 
 # Allow access to configuration files generated on the fly
-# from netplan
-/run/NetworkManager/{,**} r,
-
-# Allow writing dhcp files to a well-known system location
-/run/NetworkManager/dhcp/{,**} w,
+# from netplan and let NetworkManager store its DHCP leases
+# in the dhcp subdirectory so that console-conf can access
+# it.
+/run/NetworkManager/{,**} rw,
```

Comment:
My understanding based on IRC discussion was that we still wanted to limit write access to certain files under /run/NetworkManager/*, but allow a mkdir on /run/NetworkManager. The changes here remove the limitations on write access. I suggest instead of the above changes, add a single rule:

```
# Allow NetworkManager to create the /run/NetworkManager directory
/run/NetworkManager/ w,
```